### PR TITLE
Fix last lap records, add locales, and update `.github`

### PR DESCRIPTION
**Pull request by TonybynMp4**
**Describe Pull request**
Fixed issue https://github.com/qbcore-framework/qb-lapraces/issues/31, updates .github and adds locales

## Checklist

- [x] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -5,13 +5,13 @@ addReviewers: true
 addAssignees: author
 
 # A list of reviewers to be added to pull requests (GitHub user name)
-reviewers:
-  - /maintenance
-
-# A list of keywords to be skipped the process that add reviewers if pull requests include it
+reviewers: 
+  - QBCore-Remastered/maintenance
```
Comment (BerkieBb) [inline .github/auto_assign.yml]: Qbox*

File: fxmanifest.lua [hunk 2]
```diff
@@ -6,7 +6,11 @@ version '1.0.0'
 
 ui_page 'html/index.html'
 
-shared_script 'config.lua'
+shared_scripts {
+    '@qb-core/shared/locale.lua',
+    'locales/*.lua',
```
Comment (D4isDAVID) [inline fxmanifest.lua:12]:
```suggestion
    'locales/en.lua',
    'locales/*.lua',
```
`locales/en.lua` is also needed here.

**Review by BerkieBb — CHANGES_REQUESTED** [review]

File: server/main.lua [hunk 3]
```diff
@@ -549,34 +540,34 @@ QBCore.Commands.Add("cancelrace", "Cancel going race..", {}, false, function(sou
...
+        TriggerClientEvent('QBCore:Notify', source, Lang:t('error.notauthorized', {to = Lang:t('general.dothis')}), 'error')
     end
 end)
 
 -- Threads
 
 CreateThread(function()
     local races = MySQL.query.await('SELECT * FROM lapraces', {})
-    if races[1] ~= nil then
+    if races and races[1] ~= nil then
```
Comment (D4isDAVID) [inline server/main.lua]:
```suggestion
    if races and races[1] then
```

File: server/main.lua [hunk 4]
```diff
@@ -93,9 +93,9 @@ local function GetRaceId(name)
 end
 
 local function GenerateRaceId()
-    local RaceId = "LR-" .. math.random(1111, 9999)
+    local RaceId = "LR-" .. math.random(0000000, 9999999)
```
Comment (D4isDAVID) [inline server/main.lua]:
```suggestion
    local RaceId = "LR-" .. math.random(0, 9999999)
```
`0000000` is equivalent to `0` (or change it to `1111111` idk what's better).

File: server/main.lua [hunk 5]
```diff
@@ -93,9 +93,9 @@ local function GetRaceId(name)
...
     while Races[RaceId] ~= nil do
-        RaceId = "LR-" .. math.random(1111, 9999)
+        RaceId = "LR-" .. math.random(0000000, 9999999)
```
Comment (D4isDAVID) [inline server/main.lua]:
```suggestion
        RaceId = "LR-" .. math.random(0, 9999999)
```
Ditto

File: .github/workflows/lint.yml [hunk 6]
```diff
@@ -13,11 +13,11 @@ jobs:
         with:
           capture: "junit.xml"
           args: "-t --formatter JUnit"
-          extra_libs: mysql+polyzone+qblocales
+          extra_libs: mysql
```
Comment (D4isDAVID) [inline .github/workflows/lint.yml]:
```suggestion
          extra_libs: mysql+qblocales
```
Linter should know about locales.

File: .github/auto_assign.yml [hunk 7]
```diff
@@ -5,13 +5,13 @@ addReviewers: true
...
+  
+# A list of keywords to be skipped the process that add reviewers if pull requests include it
```
Comment (D4isDAVID) [inline .github/auto_assign.yml]:
```suggestion
reviewers:
  - QBCore-Remastered/maintenance

# A list of keywords to be skipped the process that add reviewers if pull requests include it
```
Remove trailing spaces.

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

**Pull request merged**